src/KeyBindingsDefaults.ts

```diff
@@ -77,6 +84,14 @@ const messageComposerBindings = (): KeyBinding<MessageComposerAction>[] => {
                 shiftKey: true,
             },
         },
+        {
+            action: MessageComposerAction.FormatLink,
+            keyCombo: {
+                key: Key.K,
+                altKey: !isMac,
+                shiftKey: isMac,
+            },
```

Comment:
this seems to say `Shift + K` on macOS which seems to conflict with typing an uppercase K? Presumably the ctrlOrCmd should stick around?

Reply:
Yes, you are totally right, I accidentally deleted it ... should be fixed now.